pyomo/contrib/parmest/scengennotes.txt

```diff
@@ -0,0 +1,47 @@
+DLW March 2020 parmest to mpi-sppy
```

Comment:
Do you want this file added to the repo (this looks like internal team planning notes)?

Reply:
Nope.  Thanks for helping clean up my messes.